sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/OfflineStorageTests.cs

```diff
@@ -252,7 +264,16 @@ protected override bool OnTryLease(int leasePeriodMilliseconds)
 
             protected override bool OnTryDelete()
             {
-                throw new NotImplementedException();
+                try
+                {
+                    _mockStorage.RemoveAt(0);
```

Comment:
will `_mockStorage.Remove(this)` work here?

Reply:
it does. updated.